python/tvm/relax/op/nn/nn.py

```diff
@@ -659,14 +659,15 @@ def batch_norm(
     beta: Expr,
     moving_mean: Expr,
     moving_var: Expr,
-    axis: int,
+    axis: int = 1,
```

Comment:
Is there any reason to add such default value?

Reply:
Common batch norm operators takes the axis indexed 1 as the axis to calculate mean and variance, such as torch.nn.functional.batch_norm.

Reply:
Our difference from torch is that we may have layouts like NHWC, but torch only accepts NCHW. Considering this maybe we can remove the default value 1.